DesignPattern-RequestResponse/manifest.json

```diff
@@ -0,0 +1,44 @@
+{
+    "title": "Request Response over HTTP",
+    "description": "Receive a message over HTTP, perform some other work of your choosing, and then return a response",
+    "tags": [
+        "HTTP"
+
+    ],
+    "skus": [
+        "consumption",
+        "standard"
+    ],
+    "kinds": [
+        "stateful",
+        "stateless"
+    ],
+    "details_description": "This template serves the starting point for building synchronous Request Response patterns using Logic Apps. Learn more about these capabilites here -  https://learn.microsoft.com/azure/connectors/connectors-native-reqres?tabs=consumption ",
+    "details": {
+        "By": "Microsoft",
+        "Type": "Workflow",
+        "Trigger": "Request",
+        "Category": "Design Patterns , HTTP"
```

Comment:
can we remove the gaps in between the , ? We parse those by doing split(",")
`"Category": "Design Patterns,HTTP"`